tests/test_match_data.py

```diff
@@ -18,10 +18,17 @@
 class TestMatchData:
     def setup(self):
         self.match = MatchData('name', ['one', 'two'], {'{word}': ['value', 'tokens']}, 0.5)
+        self.sentence = ["it", "'", "s", "a", "new", "sentence"]
+        self.sentence2 = ["the", "parents", "'", "house"]
 
     def test_detokenize(self):
         self.match.detokenize()
         assert self.match.sent == 'one two'
 
         correct_match = MatchData('name', 'one two', {'word': 'value tokens'}, 0.5)
         assert self.match.__dict__ == correct_match.__dict__
+
+    def test_handle_apostrophes(self):
+        joined_sentence = self.match.handle_apostrophes(self.sentence)
+        assert joined_sentence == "it's a new sentence"
+        assert joined_sentence == "the parents' house"
```

Comment:
I think before this line there should be a line

```
joined_sentence = self.match.handle_apostrophes(self.sentence2)
```

Reply:
added a second variable. You're right I missed that. Thanks